src/App.tsx

```diff
@@ -8,13 +12,18 @@ export const App = () => {
     <div data-cy="app">
       <Navbar />
 
-      <div className="section">
+      <main className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/home" element={<Navigate to="/" replace={true} />} />
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path=":slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested route configuration for '/people/:slug' might not work as expected because the parent route '/people' is not set up to render an outlet for nested routes. Consider using an `Outlet` component in the `PeoplePage` to render nested routes.